docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -63,6 +63,10 @@ Some organizations require selecting hardening rules. For example, the US govern
 
 [^CMU2018]: Carnegie Mellon University (CMU), [Top 10 Secure Coding Practices](https://wiki.sei.cmu.edu/confluence/display/seccode/Top+10+Secure+Coding+Practices), SEI CERT Coding Standards Wiki, 2018-05-02.
 
+**What should you do when compiling compilers?**
+
+If you are compiling a C/C++ compiler, where practical make the generated compiler's default options the *secure* options (as documented here). For example, when compiling GCC, use `--enable-default-pie` (which makes `-fPIE` and `-pie` the defaults) and when compiling clang on Linux systems set `CLANG_DEFAULT_PIE_ON_LINUX` (which has a similar effect).
```

Comment:
to clarify configurations vs flags:

"when compiling GCC, use the configuration `--enable-default-pie` (which enables the flags `-fPIE` and `-pie` by default)"

Reply:
Done!